reconcile/terraform_vpc_peerings.py

```diff
@@ -7,9 +7,9 @@
 
 import reconcile.utils.aws_api as awsapi
 from reconcile.utils.defer import defer
-from reconcile.utils.ocm import OCMMap
-from reconcile.utils.terraform_client import TerraformClient as Terraform
-from reconcile.utils.terrascript_client import TerrascriptClient as Terrascript
+import reconcile.utils.ocm as ocm
```

Comment:
I've noticed a pattern where we're getting rid of `from module import Class` imports.  These imports are super common in Python code bases and even suggested in [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports) except when there are name clashes.  Are these being removed because of [this section in our style guide](https://service.pages.redhat.com/dev-guidelines/docs/appsre/python-style-guide/#imports), or some other advantage that I'm missing?

Reply:
I need it to allow testslide to do its magic when it comes to mocking constructors. The way Ptyhon handles namespaces makes it impossible for testslide to reliably mock stuff that is imported into the namespace as `from foo import bar`.

Reply:
Thanks for the explanation.  So, generally speaking the `from package import Class` imports are fine as long as we avoid `__init__()` with side effects?

Reply:
Yeah, they should be fine as long as there are no side effects.